api/test/e2e/scenario-interventions/scenario-interventions.spec.ts

```diff
@@ -404,6 +405,129 @@ describe('ScenarioInterventionsModule (e2e)', () => {
       expect(newSourcingRecords.length).toBe(1);
       expect(newSourcingRecords[0].tonnage).toEqual('500');
     });
+
+    test('Create a scenario intervention of type Change of supplier location, with correct data and multiple year sourcing records should be successful', async () => {
```

Comment:
Very minor one. We are aiming to use BDD across all our testing side. Could you please refactor it to match BDD syntax?

Thanks :)